src/SFA.DAS.AssessorService.Web.Staff/Views/CertificateApprovals/_ApprovalsDetailsPartial.cshtml

```diff
@@ -1,131 +1,232 @@
-@model IEnumerable<SFA.DAS.AssessorService.Web.Staff.ViewModels.Private.CertificateDetailApprovalViewModel>
-
-<table class="table table-font-xsmall">
+﻿@using Microsoft.AspNetCore.Mvc.ModelBinding
+@using SFA.DAS.AssessorService.Domain.Consts;
+@using SFA.DAS.AssessorService.Domain.Paging
+@model SFA.DAS.AssessorService.Domain.Paging.PaginatedList<SFA.DAS.AssessorService.Web.Staff.ViewModels.Private.CertificateDetailApprovalViewModel>
+@{
+    var results = Model.Items.ToList();
+    var approvalresults = ViewData["ApprovalResults"] as SFA.DAS.AssessorService.Web.Staff.ViewModels.Private.CertificateDetailApprovalViewModel[];
+    var actionHint = ViewData["ActionHint"] as string;
+    var pageIndex = ViewData["PageIndex"] as int?;
+}
+<table class="table responsive table-font-xsmall table-spaced">
     <thead>
-    <tr>
-        <th>Apprentice</th>
-        <th>ULN</th>
-        <th>Employer</th>
-        <th>Training Provider</th>
-        <th>Date Requested</th>
-        <th>Approve/Reject</th>
-    </tr>
+        <tr>
+            <th>Apprentice</th>
+            <th>ULN</th>
+            <th>Employer</th>
+            <th>Training Provider</th>
+            @if (results[0]?.PrivatelyFundedStatus == CertificateStatus.Rejected)
+            {
+            <th>Reason For Rejection</th>
+            }
+            @if (results[0]?.PrivatelyFundedStatus != CertificateStatus.SentForApproval)
+            {
+                <th>Date Requested</th>
+            }
+            else
+            {
+                <th>Date Sent For Approval</th>
+            }
+            @if (results[0]?.PrivatelyFundedStatus != CertificateStatus.Approved)
+            {
+                <th>Action</th>
+            }
+        </tr>
     </thead>
     <tbody>
+        @for (int i = 0; i < results.Count; i++)
+        {
+            @:<tr>
+            <td data-label="Apprentice" class="nowrap">
+                <div role="button" tabindex="0" class="js-expand-table-row expand-table-row" data-expand-id="@results[i].Uln"><i class="arrow arrow-closed">►</i> <span>@results[i].FullName</span></div>
+            </td>
+            <td data-label="ULN">@results[i].Uln</td>
+            <td data-label="Employer">@results[i].ContactOrganisation</td>
+            <td data-label="Training Provider">@results[i].Ukprn</td>
+            if (results[0]?.PrivatelyFundedStatus == CertificateStatus.Rejected)
+            {
+                <td data-label="Reason For Rejection">@results[i].ReasonForChange</td>
+            }
+            if (results[0]?.PrivatelyFundedStatus != CertificateStatus.SentForApproval)
+            {
+                <td data-label="Date Requested">@results[i].CreatedDay.ToString("dd/MM/yyyy")</td>
+            }
+            else
+            {
+                <td data-label="Date Sent For Approval">@results[i].UpdatedAt?.ToString("dd/MM/yyyy")</td>
+            }
```

Comment:
This should be in the VM.  AS should the header switch above.

Reply:
See above comment